examples/docker-helloworld.yaml

```diff
@@ -30,6 +30,7 @@ gateway:
     - controller
     - registry
   container_name: gateway
+  command: /usr/bin/run_filebeat.sh
```

Comment:
I didn't remove `A8_SERVICE ` in my commits - was this broken to begin with?  Is there any CI testing for the stuff in `examples/`